index.html

```diff
@@ -291,6 +291,125 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <section id="mycontact" class="footer">
+      <h4 class="invitation">
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h4>
+
+      <form action="https://formspree.io/f/mqknkkgn" method="POST" class="form-element">
+        <ul>
+          <li class="full-name">
+            <label for="name"></label>
+            <input
+              class="text"
+              type="text"
+              id="name"
+              name="full_name"
+              placeholder="Full name"
+              minlength="3"
+              maxlength="30"
+              required
+            />
+            <img class="break" src="assets/images/Container.svg" alt="line" />
+          </li>
+
+          <li class="email">             
+            <label for="mail"></label>
+            <input
+              class="text"
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="Email address"
+              required
+            />       
+
+            <img class="break" src="assets/images/Container.svg" alt="line" />
+          </li>
+
+          <li class="blog">
+            <label for="msg"></label>
+            <textarea
+              class="area"
+              id="msg"
+              name="user_message"
+              rows="10"
+              cols="50"
+              placeholder="Enter text here"
+              minlength="7"
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
```

Comment:
- [ ] The text area according to the Figma design has a background color.

Figma Design | Your Design
------------- | -------------
![hammas-contact-review](https://user-images.githubusercontent.com/74814780/175000355-1b55a8a9-5d3f-4c76-8305-f3f811c2236d.png)  |  ![hammas-version](https://user-images.githubusercontent.com/74814780/175000470-d54d6606-3f68-412e-bf9d-0ae425d6071e.png)